presentation/common/src/test/content/Descriptor.test.ts

```diff
@@ -87,21 +88,77 @@ describe("Descriptor", () => {
       expect(descriptor).to.matchSnapshot();
     });
 
-    it("creates valid descriptorJSON from CompressedDescriptorJSON with nestedField", () => {
-      const compressedDescriptorJSON = compressDescriptorJSON(testDescriptorJSON);
-      const descriptorFromCompressedJSON = Descriptor.fromJSON(JSON.stringify(compressedDescriptorJSON));
-      const descriptor = Descriptor.fromJSON(JSON.stringify(testDescriptorJSON));
-      expect(descriptor).to.deep.eq(descriptorFromCompressedJSON);
+    it("creates valid Descriptor from CompressedDescriptorJSON", () => {
+      const categories = [createRandomCategoryJSON(), createRandomCategoryJSON(), createRandomCategoryJSON(), createRandomCategoryJSON()];
+      const ids = ["0x1", "0x2", "0x3", "0x4"];
+      const testRelatedClassInfo: RelatedClassInfoJSON<string> = {
+        sourceClassInfo: ids[1],
+        targetClassInfo: ids[2],
+        relationshipInfo: ids[3],
+        isForwardRelationship: true,
+      };
+      const fields: FieldJSON<string>[] = [
+        createRandomPrimitiveFieldJSON(categories[0]), {

```

Comment:
What's the point of adding a field that doesn't require compressing / decompressing?